core/src/main/java/hudson/model/Actionable.java

```diff
@@ -53,7 +53,7 @@
      *
      * Typed more strongly than it should to improve the serialization signature.
      */
-    private volatile CopyOnWriteArrayList<Action> actions;
+    private volatile CopyOnWriteArrayList<Action> actions = new CopyOnWriteArrayList<Action>();
```

Comment:
I am not sure it is a right change. After that it will start instantiating collections for any object it Jenkins which may have actions. It causes extra performance and memory degradation, and IMHO it is not required for any `Actionable` use-case